app/models/intake/gyr_intake.rb

```diff
@@ -441,6 +443,13 @@ class Intake::GyrIntake < Intake
   after_save_commit { SearchIndexer.refresh_filterable_properties([client_id]) }
   after_destroy_commit { SearchIndexer.refresh_filterable_properties([client_id]) }
 
+  def matching_previous_year_intakes
+    attrs = [:primary_birth_date, :hashed_primary_ssn]
+    intakes = DeduplicationService.duplicates(self, *attrs, from_scope: self.class.previous_year_completed_intakes)
+    archived_intakes = DeduplicationService.duplicates(self, *attrs, from_scope: Archived::Intake::GyrIntake2021.completed_intakes)
```

Comment:
would like a second opinion on whether this is kosher. the deduplication service is probably not meant to compare 2 different tables but in this case it works because the archived intakes table has the 2 columns that are being matched. are there any problems this could cause in the future?

Reply:
What if we renamed this service to something more accurate for how we are using it across the board like `FindMatchingIntakesService` and the duplicates method could be renamed "matches"